repos/fdbt-site/src/pages/globalSettings.tsx

```diff
@@ -56,6 +57,12 @@ const GlobalSettings = ({ globalSettingsCounts, referer }: GlobalSettingsProps):
                             description="Define certain days and time periods that your tickets can be used within"
                             count={globalSettingsCounts.timeRestrictionsCount}
                         />
+                        <SettingOverview
+                            href="/manageFareDayEnd"
+                            name="Fare day end"
+                            description="Define the time at which your fare day ends"
+                            count={globalSettingsCounts.fareDayEndSet}
```

Comment:
When the count is zero here, it has nothing displayed, which looks a little odd given we have zero's above. I think something like "Not specified" or "Uncustomised" would be good to have - worth checking with Esmond.

Reply:
Will discuss tomorrow 👍

Reply:
Updated text